ansible/roles/ocp4-workload-oadp/tasks/workload.yml

```
@@ -4,22 +4,67 @@
     oadp_state: "absent"
   when: oadp_destroy is defined and oadp_destroy | bool
 
-- name: "Set {{ oadp_namespace }} namespace state to {{ oadp_state }}"
-  k8s:
-    name: "{{ oadp_namespace }}"
-    kind: Namespace
-    state: "{{ oadp_state }}"
+- name: Set namespaces
+  tasks:
+    - name: "Set {{ oadp_namespace }} namespace state to {{ oadp_state }}"
+      k8s:
+        name: "{{ oadp_namespace }}"
+        kind: Namespace
+        state: "{{ oadp_state }}"
+    - name: "Set openshift-operators namespace"
+      k8s:
+        name: openshift-operators
+        kind: Namespace
+        state: openshift-operators
+
+- name: "Set OperatorGroup and Subscription"
+  tasks:
+  - name: "Set OADP OperatorGroup state to {{ oadp_state }}"
+    k8s:
+      state: "{{ oadp_state }}"
+      definition: "{{ lookup('template', 'oadp-operator-operatorgroup.yml.j2' ) }}"
 
-- name: "Set OADP OperatorGroup state to {{ oadp_state }}"
-  k8s:
-    state: "{{ oadp_state }}"
-    definition: "{{ lookup('template', 'oadp-operator-operatorgroup.yml.j2' ) }}"
+  - name: "Set VolSycn OperatorGroup"
+    k8s:
+      state: "{{ oadp_state }}"
+      definition: "{{ lookup('template', 'volSync-operatorgroup.yml.j2' ) }}"
```

```suggestion
      definition: "{{ lookup('template', 'volsync-operatorgroup.yml.j2' ) }}"
```
Let's not mix cases here?